src/core/thread/mle_router.cpp

```diff
@@ -2445,6 +2446,18 @@ otError MleRouter::HandleChildUpdateResponse(const Message &aMessage, const Ip6:
                      error = OT_ERROR_SECURITY);
     }
 
+    // Status
+    if (Tlv::GetTlv(aMessage, Tlv::kStatus, sizeof(status), status) == OT_ERROR_NONE)
```

Comment:
I like how this is going.  However, OT does not currently implement a Child sending an MLE Child Update Response with Status TLV.  Are you also planning to add this feature?

Reply:
Such a message was sent by other vendor's stack during certification tests. I think that currently adding the sending feature is not necessary, but it's safer to process such a message if received.